src/test/java/com/fastcampus/projectboard/util/test/FormDataEncoderTest.java

```diff
@@ -15,7 +15,7 @@
 
 @DisplayName("테스트 도구 - Form 데이터 인코더")
 @Import({FormDataEncoder.class, ObjectMapper.class})
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE, classes = Void.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE, classes = void.class)
```

Comment:
사실 `classes`의 javadoc을 읽어보시면 이 안에 들어가는 것은 component class로 정해져 있습니다.
`void.class`는 스프링 컴포넌트는 아니니까 엄밀히 말하면 정확한 사용은 아니에요.
하지만 마침 이 테스트는 이 안에 지정해 넣을 좋은 컴포넌트들을 필요로 합니다.

```suggestion
@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE, classes = {FormDataEncoder.class, ObjectMapper.class})
```

이렇게 할 경우 위에 `@Import`는 삭제해도 됩니다.